vrecord

```diff
@@ -362,6 +373,80 @@ _master_gui(){
+                                                        label="Enter the name of the capture configuration profile (Example: Umatic1)" \
+                                                        mandatory=Yes
+        _construct_pashua_id CC_DECK_MAKE               x=20 y=280 type=textbox height=30 \
+                                                        label="Deck Make" text="Enter Deck Make"
+        _construct_pashua_id CC_DECK_MODEL              x=20 y=215 type=textbox height=30 \
+                                                        label="Deck Model" text="Enter Deck Model"
+        _construct_pashua_id CC_DECK_Serial             x=20 y=150 type=textbox height=30 \
+                                                        label="Deck Serial Number" text="Enter Serial Number"
+        _construct_pashua_id CC_TBC_MAKE                x=340 y=280 type=textbox height=30 \
+                                                        label="TBC Make" text="Enter TBC Make"
+        _construct_pashua_id CC_TBC_MODEL               x=340 y=215 type=textbox height=30 \
+                                                        label="TBC Model" text="Enter TBC Model"
+        _construct_pashua_id CC_TBC_Serial              x=340 y=150 type=textbox height=30 \
+                                                        label="TBC Serial Number" text="Enter TBC Serial Number"
+        _construct_pashua_id CC_ADC_MAKE                x=670 y=280 type=textbox height=30 \
+                                                        label="Analog-Digital Converter Make" text="Enter Analog-Digital Converter Make"
+        _construct_pashua_id CC_ADC_Model               x=670 y=215 type=textbox height=30 \
+                                                        text="Enter Analog-Digital Converter Model" label="Analog-Digital Converter Model" 
+        _construct_pashua_id CC_ADC_Serial              x=670 y=150 type=textbox height=30 \
+                                                        label="Analog-Digital Converter Serial Number" text="Enter Analog-Digital Converter Serial Number"
+        _construct_pashua_id CC_NOTES                   x=20 y=15 type=textbox height=90 width=500 \
+                                                        label="Notes:"
+}
+
+_construct_capture_config_gui > "${PASHUA_CONFIGFILE}"
+_pashua_run
+if [[ "${cb}" = 0 ]] ; then
+    if  [[ ! -d "${CC_PROFILE_DIR}" ]] ; then
+        CC_PROFILE_DIR="${CC_PROFILE_DIR_Default}"
+        if [[ ! -d "${CC_PROFILE_DIR_Default}" ]] ; then
+            mkdir "${CC_PROFILE_DIR_Default}" 
+            CC_PROFILE_DIR="${CC_PROFILE_DIR_Default}"
+        fi
+    fi
+    CAPTURE_CONFIGURATION_PROFILE=""$(echo ${CC_PROFILE_DIR})"/"$(echo ${CC_PROFILE_NAME})""
+    echo
+    echo "path is "$(echo ${CC_PROFILE_DIR} | sed 's/ /\\\ /g')""
+    echo "name is ${CC_PROFILE_NAME}"
+    echo "profile is "$(echo "${CAPTURE_CONFIGURATION_PROFILE}" | sed 's/ /\\\ /g')" .txt"
```

Comment:
same as above, unneeded double quotes here (also why is there a space before `.txt`).
maybe just `echo "profile is '${CAPTURE_CONFIGURATION_PROFILE}.txt'"`